articles/openshift/cluster-administration-cluster-admin-role.md

```diff
@@ -33,6 +32,12 @@ For example, having the `list` verb means that you can display a list of all obj
 
 You can configure the customer-admin-cluster cluster role only during cluster creation by providing the flag `--customer-admin-group-id`. To learn how to configure Azure Active Directory and the Administrators group, see [Azure Active Directory integration for Azure Red Hat OpenShift](howto-aad-app-configuration.md).
 
+## Confirm membership in the customer administrator role
+
+To confirm your membership in the customer admin group, try the OpenShift CLI commands `oc get nodes` or `oc projects`. `oc get nodes` will show a list of nodes if you have the customer-admin-cluster role, and a permission error if you only have the customer-admin-project role. `oc projects` will show all projects in the cluster as opposed to just the projects you are working in.
+
+To further explore roles and permissions in your cluster, you can use the [`oc policy who-can <verb> <resource>`](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/developer-cli-commands.html#policy) command.
```

Comment:
It would be good to use 3.11 link for this as this ARO version is using 3.11 release payload. And this will not change

Reply:
Maybe this? https://docs.openshift.com/container-platform/3.11/admin_guide/manage_rbac.html#managing-role-bindings

Reply:
Sure thing! I'll update that link.